client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -199,7 +199,19 @@ export const BatchTable: FC<
       getIsError: rowData =>
         errorsContext.getError(rowData)?.__typename ===
         'StockLineReducedBelowZero',
-      Cell: props => <NumberInputCell {...props} decimalLimit={2} min={0} />,
+      Cell: props => {
+        const value = props.rowData.countedNumberOfPacks;
+        return (
+          <NumberInputCell
+            {...props}
+            decimalLimit={2}
+            min={0}
+            // "Formatting" changes null/undefined (uncounted) value to 0
+            // but we want uncounted to render an empty input cell
+            noFormatting={value === null || value === undefined}
```

Comment:
This shouldn't be necessary with the fixes done to `NumericTextInput` that are in `develop`. (And if it's not, we should fix it in that component rather than misusing the `noFormatting` prop here -- but I'm 90% sure it is fixed)

Reply:
I'm tempted to suggest you copy and paste the whole `NumericTextInput` component from `develop` into this branch so we can have those fixes in 2.0.0 as well? What do you think? Maybe at least try it out to confirm that it fixes this problem here?

Reply:
Just pulled the latest develop in (this branch is going into develop) but not resolved there... will have a play moving this into that component though!

Reply:
Do you this it's behaviour we would want for all NumericTextInputs? That if the value is undefined the input renders empty? Or some additional flag?

Reply:
> That if the value is undefined the input renders empty?

Yes, that's the intended behaviour

Reply:
Oh found it! It was handling undefined, but not null 👍

Reply:
Yeah, React doesn't like `null` as a value in form components -- that's what often gives that "changing from controlled to uncontrolled component" error.